README.md

```diff
@@ -1,178 +1,121 @@
-<!-- BEGIN MODULE HOOK -->
+# IBM Cloud Databases for etcd module
 
-<!-- Update the title to match the module name and add a description -->
-# Terraform Modules Template Project
-<!-- UPDATE BADGE: Update the link for the following badge-->
-[![Incubating (Not yet consumable)](https://img.shields.io/badge/status-Incubating%20(Not%20yet%20consumable)-red)](https://terraform-ibm-modules.github.io/documentation/#/badge-status)
+[![Graduated](<https://img.shields.io/badge/Status-Graduated%20(GA%20candidate)-green>)](https://github.ibm.com/GoldenEye/documentation/blob/master/status.md)
```

Comment:
The Goldeneye references. you can use public redis module as a reference.

Reply:
Removed the goldeneye reference.